src/expandableCalendar/index.tsx

```diff
@@ -539,6 +539,7 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
           onDayPress={_onDayPress}
           accessibilityElementsHidden // iOS
           importantForAccessibility={'no-hide-descendants'} // Android
+          disableDaySelection={!!numberOfDays && numberOfDays > 1}
```

Comment:
Why this condition can't be in the WeekCalendar instead of passing it from the ExpandableCalendar? This way 'disableDaySelection' won't be a public prop but an internal one, that the WeekCalendar can pass the Week component (which is a private component).